app/auth/SignUp.tsx

```diff
@@ -0,0 +1,311 @@
+                placeholder='Password'
+                placeholderTextColor="#888"
+                secureTextEntry={true}
+                onChangeText={(text) => setPassword(text)}
+              />
+        
+
+              {/* Confirm Password */}
+              <Text style = {styles.titleinput}>Confirm Password *</Text>
+              <TextInput
+                style = {confirmPassword.length == 0 ||password == confirmPassword ? styles.input : styles.inputWrong}
+                placeholder= 'Confirm Password'
+                placeholderTextColor="#888"
+                secureTextEntry={true}
+                onChangeText={(text) => setConfirmPassword(text)}
+              />
+
+              {/* Register Button */}
+              <TouchableOpacity style={styles.buttonStrive} onPress={() => onClickStrive(password, confirmPassword, name, surname, email)}>
+                <Text style={styles.buttonText}>Strive with us</Text>
+              </TouchableOpacity>
+
+          
+              {/* OR */}
+              <Text style = {styles.or}> 
+                  OR
+              </Text>
+
+              {/* Sign Up buttons for Google */}
+              <TouchableOpacity style={styles.buttonContinueWith} onPress={() => alert('Sign In with Google')}>
+                <View style={styles.buttonIcon}>
+                    <Image source={require('@/assets/images/sign-up-screen/google.png')} style={styles.icon} />
+                    <Text style={styles.buttonText}>Continue with Google</Text>
+                </View>
+              </TouchableOpacity>
+
+              {/* Sign Up buttons for Facebook */}
+              <TouchableOpacity style={styles.buttonContinueWith} onPress={() => alert('Sign In with Facebook')}>
+                <View style={styles.buttonIcon}>
+                    <Image source={require('@/assets/images/sign-up-screen/facebook.png')} style={styles.icon} />
+                    <Text style={styles.buttonText}>Continue with Facebook</Text>
+                </View>
+              </TouchableOpacity>
+              
+
+              </View>
+          
+          </View>
+
+        </ScrollView>
+
+      </View>
+    );
+  }
+
+    
+
+/*** 
+ * Function to check if the email is valid
+ * @param email - email to be checked
+ * @returns - true if the email is valid, false otherwise
+ */
+function isValidEmail(email : string) {
+  let reg = /^[a-zA-Z0-9]+([._-][a-zA-Z0-9]+)*@[a-zA-Z0-9]+([.-][a-zA-Z0-9]+)*\.[a-zA-Z]{2,}$/;
+  return reg.test(email);
+}
+
+/**
+ * Function to check the validity of the input fields and alert the user if there is an error
+ * @param password 
+ * @param confirmPassword 
+ * @param name 
+ * @param surname 
+ * @param email 
+ * @returns
+ */
+function onClickStrive(password : string, confirmPassword : string, name : string, surname : string, email : string) {
+  if (name.length == 0) {
+    alert('Name cannot be empty');
+}
+
+else if (surname.length == 0) {
```

Comment:
You may want to use a formatter, for which the configuration would be committed on the repository for consistency across the team.